indico/modules/events/timetable/legacy.py

```diff
@@ -51,7 +51,7 @@ def serialize_timetable(self, event, days=None, hide_weekends=False):
                 continue
             if not entry.can_view(session.user):
                 continue
-            data = self.serialize_timetable_entry(entry)
+            data = self.serialize_timetable_entry(entry, load_children=False)
```

Comment:
he actually had it like that before and i suggested keeping it the query that gets everything to avoid a mess with lazyloading and query options to avoid spamming extra queries for the child entries